hostile/1.1.0/Dockerfile

```diff
@@ -0,0 +1,71 @@
+# Set global variables
+ARG HOSTILE_VER="1.1.0"
+
+# Stage 1: Build Dockerfile
+FROM ubuntu:focal AS builder
```

Comment:
setting this as ubuntu:jammy might be helpful for you

Reply:
Made all the changes you suggested besides this one.  For some reason i was getting a different error when using `ubuntu:jammy`, so just switched it back but if its a good switch you think I'm more than willing to go through and fix the error, but wasn't sure if it was worth it or not in terms for the docker build.